clubs_guides/views.py

```diff
@@ -1,3 +1,73 @@
-from django.shortcuts import render
+from rest_framework.generics import ListAPIView, RetrieveAPIView
 
-# Create your views here.
+from clubs_guides.models import ClubsGuide, Category
+from clubs_guides.serializers import (
+    ClubsGuideSerializer,
+    CategorySerializer,
+    CategoryWithClubsGuideSerializer,
+)
+
+
+class ClubsGuidesListView(ListAPIView):
+    """
+    A view that permits a GET to allow listing all the clubs guides
+    in the database
+
+    **Route** - `/clubsguides`
+
+    """
+    queryset = ClubsGuide.objects.all()
+    serializer_class = ClubsGuideSerializer
+
+
+class ClubsGuideView(RetrieveAPIView):
+    """
+    A view that permits a GET to allow listing of a single clubs guide
+    by providing its id.
+
+    **Route** - `/clubsguides/:id`
```

Comment:
why is this the route? what ensures that this code comment does not go stale wrt the actual code behaviour?